var/spack/repos/builtin/packages/h5hut/package.py

```diff
@@ -6,24 +6,33 @@
 from spack.package import *
 
 
-class H5hut(AutotoolsPackage):
+class H5hut(AutotoolsPackage, CMakePackage):
     """H5hut (HDF5 Utility Toolkit).
     High-Performance I/O Library for Particle-based Simulations."""
 
     homepage = "https://amas.psi.ch/H5hut/"
-    url = "https://amas.web.psi.ch/Downloads/H5hut/H5hut-2.0.0rc3.tar.gz"
+    url = "https://amas.web.psi.ch/Downloads/H5hut/H5hut-2.0.0rc7.tar.gz"
     git = "https://gitlab.psi.ch/H5hut/src.git"
+    maintainers("biddisco")
 
-    version("2.0.0rc3", sha256="1ca9a9478a99e1811ecbca3c02cc49258050d339ffb1a170006eab4ab2a01790")
```

Comment:
I'm not sure if this should go through a `deprecated=true`? For rcs in normal situations I would probably not care, but given that the rcs in this case seem to be practically normal version numbers maybe it should be kept? I don't feel strongly about this though.

Reply:
There are only a couple of users of h5hut at psi and cscs. nobody is going to miss rc3

Reply:
I had the same thought...and I came to the same non-conclusion. In theory we should go for deprecating it and eventually remove it in a next release of spack, but in that case we should change the other constraint accordingly (i.e. as I was suggesting initially `@:2.0.0rc6`).

Reply:
Thanks ok, just wanted to check if removing the version is going to break anybody's workflow. I've nothing against removal in that case.